index.html

```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
+            
+        </div>
+
+        <!-- card ends here -->
+
+
+        <!-- About session starts -->
+        <section class="about">
+            <h2 class="intro">
+                About
+                Myself
+            </h2>
+    
+            <p class="about-me-details">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+    
+    
+    
+            <h4 class="social-media-intro">LET’S CONNECT</h4>
+    
+            
+                <ul class="social-media-icon">
+                   <li class="social-media"><a class="list-style" href="#" ><img src="tweeter.svg"        alt="" ></a></li>
+                   <li class="social-media"><a class="list-style" href="#" ><img src="Linkedin-icon.svg"  alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style " href="#" ><img src="mediaThree.svg"     alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style" href="#" ><img src="mediaFour.svg"      alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style" href="#" ><img src="mediaFive.svg"      alt=""></a></li>
+    
+                </ul>
+
+                <button type="submit"  class="btnNormal">Get my resume</button>
+
+                <ul class="lang-sec">
+                    <li class="lang-tille-link">
+                    <h4 class="lang-title">Languages</h4>
+                    <a class="" href="#">
+                    <img class="language-display-icon" src="lanDisplayIcon.svg" alt="">
+                    </a>
+     
+                        </li>
+                    </ul>
+    
+                <ul class="lang-icon">
+                    <li class="langIconDetails first-lang">
+                     <img class="iconDetailsImg" src="javascriptImage.svg" alt="">
+                    <a class="langIconDetailsLink" href="#"><h4 class="lang-icon-">JavaScript</h4></a>   
+                    </li>
+                    <li class="langIconDetails sec-lang">
+                    <img class="iconDetailsImg" src="htmlImage.svg" alt="">
+                    <a class="langIconDetailsLink" href="#"><h4 class="lang-icon-">HTML</h4></a>   
+                    </li>
+                    <li class="langIconDetails thd-lang">
+                    <img class="iconDetailsImg" src="cssImage.svg" alt="">
+                    <a class="langIconDetailsLink" href="#"><h4 class="lang-icon-">CSS</h4></a>   
+                     </li>
+                </ul>
+
+                <ul class="lang-sec">
+                    <li class="lang-tille-link">
+                    <h4 class="lang-title">Frameworks</h4>
+                    <a class="" href="#"><img class="language-display-icon" src="lanDisplayIcon.svg" alt=""></a>
```

Comment:
- [ ] Please align the items in the language , frameworks , sections with flex box and make sure they looks exactly like Figma design 



| Your design | Figma |
| ------------- | ------- | 
| ![image](https://user-images.githubusercontent.com/74744787/164708340-f439e6e5-ec95-4ec6-8e1c-bf4b9720d1b7.png) | ![image](https://user-images.githubusercontent.com/74744787/164708419-ec6a7341-57e7-4ddb-85db-7af17c12e4bb.png) |

Please fix the arrows here and make sure that the distance between the arrow and the title is the same as in Figma design